src/ray/gcs/format/gcs.fbs

```diff
@@ -119,12 +119,18 @@ table FunctionTableData {
 table ObjectTableData {
   // The size of the object.
   object_size: long;
+  // Is object in-lined?
```

Comment:
Add a description of what inlined means and which fields will be set if this flag is true.